core/node/proof_data_handler/Cargo.toml

```diff
@@ -24,12 +24,12 @@ axum.workspace = true
 tokio.workspace = true
 tower-http = { workspace = true, features = ["compression-zstd", "decompression-zstd"] }
 tracing.workspace = true
+chrono.workspace = true
 
 [dev-dependencies]
 hyper.workspace = true
-chrono.workspace = true
 zksync_multivm.workspace = true
 serde_json.workspace = true
 tower.workspace = true
-zksync_basic_types.workspace = true
 zksync_contracts.workspace = true
+zksync_basic_types.workspace = true
```

Comment:
Nit: You don't usually need `zksync_basic_types` as a direct dep if you depend on `zksync_types`; the latter re-exports a substantial part of basic types.

Reply:
Fixed [here](https://github.com/matter-labs/zksync-era/pull/2980/commits/16a0e555a1cf746bcbf2e62dac00122132d4ea0e). ✅